.github/CONTRIBUTING.md

```diff
@@ -1,134 +1,93 @@
+-   [Jordan Eisenburger](https://github.com/Jordaneisenburger)
 
-Any and all community participation in this backlog is encouraged and appreciated.
-Even foundational infrastructure stories are available for a generous developer to take on.
+**Community Advisory Board**
+: Consists of Community Maintainer(s) and Core Team members who help drive prioritization and scoping for community driven **help wanted** features and enhancements.
 
-To get started, look for issues tagged with the **[help wanted]** labels.
-These issues are ready for community ownership.
+**Directly Responsible Individual**
+: The assigned developer for a specific issue.
+This person is responsible for ensuring the issue is completed in a reasonable amount of time and to a certain standard.
 
-### Claiming an issue on the roadmap
+## Ways to contribute
 
-If you are interested in taking ownership of a roadmap feature or issue, we ask that you go through the following process.
-This helps us organize and forecast the progress of our projects.
+-   [Identify and create issues](#identify-and-create-issues)
+-   [Select a groomed issue](#select-a-groomed-issue)
+-   [Help answer community questions](#help-answer-community-questions)
 
-#### Step 1: Add an issue comment
+### Identify and create issues
 
-Add a comment on an issue expressing your interest in taking ownership of it.
-Make sure your GitHub profile includes an email address so we can contact you privately.
+If you encounter an issue while using PWA Studio or you have a suggestion for a new feature, let us know by [creating an issue][].
+Provide as much detail as you can to help us reproduce or analyze the issue before prioritizing it.
 
-#### Step 2: Meet with a maintainer
+When you create an issue, add the **help wanted** label to bring it to the attention of the Community Advisory Board.
+When the issue is created, it is placed in the **Backlog** column of the [Community Backlog][] project.
 
-A maintainer will contact you and ask to set up a real-time meeting to discuss the issue you are interested in owning.
-This meeting can be in person, video chat, audio chat, or text chat.
+_Please avoid creating GitHub issues asking for help on bugs in your own project that are outside the scope of this project._
 
-In general this meeting is brief but can vary with the complexity of an issue.
-For larger issues, we may schedule follow-up meetings.
+#### Grooming new issues
 
-During this meeting, we provide you with any additional materials or resources you need to work on the issue.
+Every week, the Community Advisory Board meets to look at the **Backlog** column of the Community Backlog project.
+The board grooms the backlog issues and moves issues that are well-defined and has value to the **Prioritized** column.
 
-#### Step 3: Provide an estimate
+If an issue does not provide enough details or provides no value, it is closed.
```

Comment:
@jcalcaben If we feel that an issue does not provide enough detail I think we shouldn't close it but rather leave a comment and give them the opportunity to respond if there is no reaction before the next grooming session we should indeed close it.

Reply:
Updated statement